<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1394</width>
    <height>593</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : #0C3A7A;
color : white</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="horizontalGroupBox">
      <property name="title">
       <string>이미지</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="Monitor_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>448</width>
           <height>336</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>448</width>
           <height>336</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color : rgb(100,100,100);
</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Monitor_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>448</width>
           <height>336</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>448</width>
           <height>336</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color : rgb(100,100,100);
</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Monitor_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>448</width>
           <height>336</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>448</width>
           <height>336</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color : rgb(100,100,100);
</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="horizontalGroupBox_2">
      <property name="title">
       <string>Kr</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
background-color : rgb(255,255,255);
color : rgb(0,0,0);
border-radius: 20px;
text-align:left;
padding : 7px;
border: 3px solid rgb(0,0,0)

</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">
background-color : rgb(255,255,255);
color : rgb(0,0,0);
border-radius: 20px;
text-align:left;
padding : 7px;
border: 3px solid rgb(0,0,0)

</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">
background-color : rgb(255,255,255);
color : rgb(0,0,0);
border-radius: 20px;
text-align:left;
padding : 7px;
border: 3px solid rgb(0,0,0)

</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="verticalGroupBox">
      <property name="title">
       <string>Choice</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="LEFT">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 #52E5E7, stop:1 #130CB7);
	border-radius: 22px;
	border: 2px solid rgb(19, 50, 92)
}
QPushButton:pressed {
background:rgb(0,0,0);
color : rgb(255,255,255)
}</string>
         </property>
         <property name="text">
          <string>LEFT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="MID">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: #FF3A85;
	background-color: white;
	border: 5px solid #FF3A85;;
	border-radius: 20px;
}

QPushButton:pressed {
background:rgb(0,0,0);
color : rgb(255,255,255)
}</string>
         </property>
         <property name="text">
          <string>unknown</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RIGHT">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 #52E5E7, stop:1 #130CB7);
	border-radius: 22px;
	border: 2px solid rgb(19, 50, 92)
}
QPushButton:pressed {
background:rgb(0,0,0);
color : rgb(255,255,255)
}</string>
         </property>
         <property name="text">
          <string>RIGHT</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1394</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
